### ISSUE TITLE
Possible way to set intersection to a different character for just one row/column?

### ISSUE THREAD

**Issue by stefan-mysten** [zhiburt/tabled#359]
Hi there! 

I was wondering if it is possible to customize the table to not have the ┰ at the top intersection of the first column with second column on the first row, and instead to simply have the ─ character as the rest. Similar for the bottom intersection. I tried playing with the papergrid low-level API, Border char, Verticals, etc, but I cannot figure out how to only tell it that I want to remove / change the vertical for one row / column. The style I am using is modern, and on top of my rows I add a header with Panel::header. The table has no set headers. 

Basically, from this --> 
![image](https://github.com/zhiburt/tabled/assets/135084671/2af35fd6-47d0-4341-9252-aaba00416b6f)

I would like to have this (ignore the change in color) - so no intersections.
![image](https://github.com/zhiburt/tabled/assets/135084671/31b7fc35-cd23-45df-ba7c-6ae9a452cc21)

**Comment by stefan-mysten** [comment]
I managed to figure it out after spending a bit of time thinking how to do it! 
Here's the code
```
t.with(Modify::new(Cell::new(0, 0)).with(Border::default().corner_bottom_right('┬')));
t.with(Modify::new(Cell::new(0, 0)).with(Border::default().corner_top_right('─')));
```

**Comment by zhiburt** [comment]
Hi @stefan-mysten 

Sorry for a delay.

Yes you can do what you did;

But if your goal to omit unnessary symbols when you using `Span`/`Panel`.
You could try the following instead.

```
.with(tabled::settings::style::BorderSpanCorrection)
```

Take care.

**Comment by zhiburt** [comment]
You could also do

```rust
.with(
            Style::modern()
                .horizontals([HorizontalLine::new(1, Style::modern().get_horizontal()).intersection(None)]),
 )
```

**Comment by stefan-mysten** [comment]
Lovely, thank you for the alternative!
